src/index.html

```diff
@@ -100,18 +100,7 @@ <h2>Select a problem and try out Benetech's Math Editor (alpha)!</h2>
             <div id="ProblemList"></div> <!-- place for list of problems -->
         </nav>
         <main id="MainWorkArea" class="editorAndHistoryWrapper">
-            <div class="problemHeader">
-                <div>
-                    <span class="problemTitle" id="ProblemTitle" role="heading" aria-level="1">Problem title</span>
-                    <span class="problemTitle" id="ProblemMath">Problem</span>
```

Comment:
Maybe we could pass an equation here as an parameter? What do you think?
https://reactjs.org/docs/components-and-props.html

Reply:
Thanks, I'll try to do this in a new PR.